lms/services/dashboard.py

```diff
@@ -88,6 +90,26 @@ def delete_dashboard_admin(self, dashboard_admin_id: int) -> None:
         """Delete an existing dashboard admin."""
         self._db.query(DashboardAdmin).filter_by(id=dashboard_admin_id).delete()
 
+    def get_request_admin_organizations(self, request) -> list[Organization]:
```

Comment:
Moved here from the views constructor: get the admins for the current user or just the one organization that the staff member is currently interested in.